modules/customizing-parameters-serving-runtime.adoc

```diff
@@ -0,0 +1,35 @@
+:_module-type: PROCEDURE
+
+[id="customizing-parameters-serving-runtime{context}"]
+= Customizing the parameters of a deployed model-serving runtime
+
+[role='_abstract']
+You can modify the parameters of an existing runtime to suit your business requirements.
+
+NOTE: Customizing the parameters of a runtime only affects the model deployment selected.
+
+.Prerequisites
+* You have logged in to {productname-short} as a user with {productname-short} administrator privileges.
+* You have deployed a model-serving runtime on the single model-serving platform.
+
+.Procedure
+. From the {productname-short} AI dashboard, click *Data Science Projects* in the left menu.
++
+The *Data Science Projects* page opens.
+. Click the name of the project that you want to deploy a model in.
++
+A project details page opens.
+. Click the *Models* tab.
+. Click the action menu (⋮) and select *Edit*.
+. Customize the runtime parameters in the *Configuration parameters* section:
+.. Modify the values in *Additional serving runtime arguments* to define how the deployed model behaves.
+.. Modify the values in *Additional environment variables* to define variables in the model's environment.
++
+NOTE: Do not modify the `port` or `-–model` parameters, because they require specific values to be set. Overwriting these parameters can cause errors.
```

Comment:
Edit: `Do not modify the `port` or `-–model` arguments, because they require specific values to be set. Overwriting these parameters can result in a failed deployment.`

This is the help text we show on the dashboard.
<img width="636" alt="Screenshot 2024-11-25 at 5 30 08 PM" src="https://github.com/user-attachments/assets/3cbd067c-7306-496d-b61a-3fffd3ed4c02">